### PR TITLE
Improve comments for docs. Add better default spacing for Inline

### LINKED ISSUE #7592: User testing sessions 4 / 5 notes
- [x] https://github.com/Shopify/polaris/pull/7593
- [x] https://github.com/Shopify/polaris/pull/7625
- [x] https://github.com/Shopify/polaris/pull/7633
- [x] https://github.com/Shopify/polaris/pull/7644
- [x] Tiles example is visually distracting https://github.com/Shopify/polaris/pull/7569
- [ ] Allow single values to be passed in place of responsive props
- [ ] Simplify code in examples so less important placeholder is farther down the page
- [ ] Allow `Inline` to render as inline element. i.e. `span` –OR– allow other display types i.e. `inline-flex` `@default 'flex'`

### PR DESCRIPTION
**Pull request by kyledurand**
Part of user testing fixes: https://github.com/Shopify/polaris/issues/7592

### CODE REVIEW

**Review by aveline — APPROVED** [review]

File: polaris-react/src/components/AlphaStack/AlphaStack.tsx [hunk 1]
```diff
@@ -17,15 +17,21 @@ type Element = 'div' | 'ul' | 'ol' | 'fieldset';
...
   children?: React.ReactNode;
-  /** Adjust vertical alignment of elements */
+  /** The vertical alignment of elements
+   * @default 'start'
+   */
   align?: Align;
   /** Toggle elements to be full width */
   fullWidth?: boolean;
-  /** Adjust spacing between elements */
+  /** The space between elements
```
Comment (kyledurand) [inline polaris-react/src/components/AlphaStack/AlphaStack.tsx]:
```suggestion
  /** The spacing between elements
```

File: polaris-react/src/components/AlphaCard/AlphaCard.tsx [hunk 2]
```diff
@@ -17,8 +17,15 @@ type CardBackgroundColorTokenScale = Extract<
 export interface AlphaCardProps {
   /** Elements to display inside card */
   children?: React.ReactNode;
+  /** Background color
+   * @default 'surface'
+   */
   background?: CardBackgroundColorTokenScale;
+  /** Spacing around the card
```
Comment (kyledurand) [inline polaris-react/src/components/AlphaCard/AlphaCard.tsx]:
```suggestion
  /** The spacing around the card
```

File: polaris-react/src/components/Columns/Columns.tsx [hunk 3]
```diff
@@ -17,7 +17,9 @@ type Spacing = {
 };
 
 export interface ColumnsProps {
-  /** The space between columns */
+  /** The space between columns
```
Comment (kyledurand) [inline polaris-react/src/components/Columns/Columns.tsx]:
```suggestion
  /** The spacing between columns
```

File: polaris-react/src/components/Inline/Inline.tsx [hunk 4]
```diff
@@ -19,7 +19,9 @@ export interface InlineProps {
   children?: React.ReactNode;
   /** Wrap stack elements to additional rows as needed on small screens (Defaults to true) */
   wrap?: boolean;
-  /** Adjust spacing between elements */
+  /** The space between elements
```
Comment (kyledurand) [inline polaris-react/src/components/Inline/Inline.tsx]:
```suggestion
  /** The spacing between elements
```

**Review by chazdean — APPROVED** [review]
Nice! ✨

**Comment by chazdean** [comment]
Also just wanted to give a heads-up that I went ahead and merged my `Inline` [PR](https://github.com/Shopify/polaris/pull/7563). But like you mentioned it should just be a minor conflict 👍🏾

**Pull request merged**